src/screens/UserManagement/CreateCustomRole.res

```diff
@@ -0,0 +1,153 @@
+module RenderCustomRoles = {
+  @react.component
+  let make = (~heading, ~description, ~groupName, ~groupsAdded, ~setGroupsAdded) => {
+    // TODO : Refactor to use in the same form
+    let (checkboxSelected, setCheckboxSelected) = React.useState(_ =>
+      groupsAdded->Array.includes(groupName)
+    )
+
+    let handleRemoveOrAdd = () => {
+      if !(groupsAdded->Array.includes(groupName)) {
+        groupsAdded->Array.push(groupName)
+        setGroupsAdded(_ => groupsAdded)
+      } else {
+        let filteredValue = groupsAdded->Array.filter(value => {value !== groupName})
+        setGroupsAdded(_ => filteredValue)
+      }
+    }
+
+    <div className="flex justify-between items-center">
+      <div className="flex flex-col gap-3 items-start col-span-1">
+        <div className="font-semibold"> {heading->React.string} </div>
+        <div className="text-base text-hyperswitch_black opacity-50 flex-1">
+          {description->React.string}
+        </div>
+      </div>
+      <div onClick={_ => handleRemoveOrAdd()}>
+        <CheckBoxIcon
```

Comment:
move the checkbox to the left side
<img width="1159" alt="image" src="https://github.com/juspay/hyperswitch-control-center/assets/120017870/7ba7ab61-6131-4a8a-ba8b-144c40fc3220">